src/gui/UserStatusSelector.qml

```diff
@@ -19,103 +19,182 @@ import QtQuick.Controls 2.15
 import QtQuick.Window 2.15
 import com.nextcloud.desktopclient 1.0 as NC
 
+import Style 1.0
+
 ColumnLayout {
     id: rootLayout
     spacing: 0
     property NC.UserStatusSelectorModel userStatusSelectorModel
 
     Label {
-        Layout.topMargin: 16
-        Layout.leftMargin: 8
-        Layout.rightMargin: 8
-        Layout.bottomMargin: 8
+        Layout.topMargin: Style.standardSpacing * 2
+        Layout.leftMargin: Style.standardSpacing
+        Layout.rightMargin: Style.standardSpacing
+        Layout.bottomMargin: Style.standardSpacing
         Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
         font.bold: true
         text: qsTr("Online status")
+        color: Style.ncTextColor
     }
         
     GridLayout {
-        Layout.margins: 8
+        id: topButtonsLayout
+
+        Layout.margins: Style.standardSpacing
         Layout.alignment: Qt.AlignTop
         columns: 2
         rows: 2
-        columnSpacing: 8
-        rowSpacing: 8
+        columnSpacing: Style.standardSpacing
+        rowSpacing: Style.standardSpacing
 
-        Button {
-            Layout.fillWidth: true
+        property int maxButtonHeight: 0
+        function updateMaxButtonHeight(newHeight) {
+            maxButtonHeight = Math.max(maxButtonHeight, newHeight)
+        }
+
+        UserStatusSelectorButton {
             checked: NC.UserStatus.Online == userStatusSelectorModel.onlineStatus
             checkable: true
             icon.source: userStatusSelectorModel.onlineIcon
             icon.color: "transparent"
             text: qsTr("Online")
             onClicked: userStatusSelectorModel.setOnlineStatus(NC.UserStatus.Online)
-            implicitWidth: 100
-        }
-        Button {
+
             Layout.fillWidth: true
+            implicitWidth: 200 // Pretty much a hack to ensure all the buttons are equal in width
```

Comment:
we already were doing it before for the exact same reason

Reply:
not sure if the comment only on this line is really helpful
if one is reading line `78` there is no comment but still the use if `implicitWidth`

Reply:
Yeah, the main change here is an explainer comment and making them slightly wider. Will add comments to each implicitWidth

Reply:
Done